inference-engine/src/inference_engine/CMakeLists.txt

```diff
@@ -160,7 +161,7 @@ if (TBBBIND_2_4_FOUND)
 endif()
 
 target_link_libraries(${TARGET_NAME} PRIVATE pugixml::static openvino::itt ${CMAKE_DL_LIBS} Threads::Threads
-                                             ngraph inference_engine_transformations)
+                                             ngraph ngraph::frontend_manager inference_engine_transformations)
```

Comment:
maybe we need to make static linkage for frontend_manager inside IE?

CC @ilyachur @slyalin

Reply:
I don't have much preferences here. But if user want to directly instantiate FEs, FEM will be used directly and it may result into two FEMs linked (one statically and another one dynamically). Is it a big deal?

Reply:
Having static and dynamic versions of FEM is not a problem, IE will not expose any FEM symbols and they will not be any conflicts.

I just wonder that it's 100 KB library and it cannot be directly injected into IE instead of adding one more dependency...

Reply:
If that is considered important, lets do this as a separate PR